src/cognito-verifier.ts

```diff
@@ -221,6 +221,8 @@ export class CognitoJwtVerifier<
   },
   MultiIssuer extends boolean,
 > extends JwtVerifierBase<SpecificVerifyProperties, IssuerConfig, MultiIssuer> {
+  private static userPoolIdRegex =
```

Comment:
If we use upper case for `JWT_REGEX` in jwt.ts, why is this not `USER_POOL_ID_REGEX` so more obvious a constant when used?

```javascript
const match = userPoolId.match(this.USER_POOL_ID_REGEX);
```

Reply:
Let's do that 👍🏻 and be consistent